llvm/lib/AsmParser/LLParser.cpp

```diff
@@ -4245,6 +4250,19 @@ bool LLParser::ParseMDField(LocTy Loc, StringRef Name, LinkageField &Result) {
   return false;
 }
 
+template <>
+bool LLParser::ParseMDField(LocTy Loc, StringRef Name, VisibilityField &Result) {
+  unsigned Res;
+  if (Lex.getKind() != lltok::kw_default && Lex.getKind() != lltok::kw_hidden &&
+      Lex.getKind() != lltok::kw_protected)
```

Comment:
Can we avoid calling `Lex.getKind()` three times?

Reply:
Thanks, Fixed it.